src/flag_gems/runtime/config.py

```diff
@@ -0,0 +1,115 @@
+import copy
+
+import triton
+
+from . import backend
+from .device import device
+
+
+class Config:
+    def __init__(self):
+        self.config = self.get_vendor_tune_config()
```

Comment:
If it is just some raw config that need to be preprocessed, do not expose it or do not use such a PLAIN name, which is confusing.